src/varnish.c

```diff
@@ -478,6 +496,27 @@ static int varnish_monitor(void *priv,
     else if (strcmp(name, "sess_fail") == 0)
       return varnish_submit_derive(conf->instance, "session",
                                    "total_operations", "failed", val);
+#if HAVE_VARNISH_V6
+    else if (strcmp(name, "sess_fail_econnaborted") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failed_econnaborted",
+                                   val);
+    else if (strcmp(name, "sess_fail_eintr") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failed_eintr", val);
+    else if (strcmp(name, "sess_fail_emfile") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failed_emfile", val);
+    else if (strcmp(name, "sess_fail_ebadf") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failedi_ebadf", val);
```

Comment:
Typo, surplus "i"

Reply:
good catch ! Should I squash the fix ?

Reply:
Looks like you already did this, thanks!